### PR TITLE
Merging the ASB patches for May 2023

### PR DESCRIPTION
**Pull request by alavalas1**
Merging the ASB patches for May 2023.

Tracked-On: OAM-109745
Signed-off-by: Reddy, Alavala Srinivasa [alavala.srinivasa.reddy@intel.com](mailto:alavala.srinivasa.reddy@intel.com)
_____________________________________________________________________________

### CODE REVIEW

**Review by tanujtekriwal — APPROVED** [review]
LGTM

File: aosp_diff/preliminary/build/make/0004-Update-security_patch_level-string.patch [hunk 1]
```diff
@@ -20,7 +20,7 @@ index 339d36bb5c..85b5300b2d 100644
      #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
      #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
 -    PLATFORM_SECURITY_PATCH := 2023-02-05
-+    PLATFORM_SECURITY_PATCH := 2023-04-01
++    PLATFORM_SECURITY_PATCH := 2023-05-01
```
Comment (shyjumon-n) [inline aosp_diff/preliminary/build/make/0004-Update-security_patch_level-string.patch:23]: Change the string only after concluding on the left out patches.

Comment (alavalas1) [reply aosp_diff/preliminary/build/make/0004-Update-security_patch_level-string.patch:23]: Left patches are added

**Review by tanujtekriwal — APPROVED** [review]
LGTM

**Review by shyjumon-n — APPROVED** [review]
Google security patch integration.
LGTM

**Pull request merged**